superset/assets/src/query/buildQueryContext.ts

```diff
@@ -1,13 +1,13 @@
-import buildDatasource, { Datasource } from './datasourceBuilder';
+import buildDatasource, { Datasource } from './buildDatasource';
+import buildQueries, { QueryObject } from './buildQueryObject';
```

Comment:
Is it `buildQueries` or `buildQueryObject`?

Reply:
`buildQueries` is probably better since we'll always return an array. I'll rename the file and the default export.